exonum-java-binding-core/src/main/java/com/exonum/binding/crypto/Crypto.java

```diff
@@ -0,0 +1,14 @@
+package com.exonum.binding.crypto;
+
+import com.goterl.lazycode.lazysodium.interfaces.Sign;
+
+public class Crypto {
```

Comment:
I'd move to `CryptoFunctions` because it is confusing to have two public classes.